docs/front-end/offline.md

```diff
@@ -0,0 +1,66 @@
+# Offline
+
+Offline could mean mulitple things in the context of Web Applications and it is important to be prepared for those different scenarios.
+
+* No Network Connectivity - Truly Offline
+* Limited/Spotty Network Connectivity - Also Known as Lie-Fi
+
+Having no network connectivity is usually what is associated with the term offline, but having limited connectivity(Lie-Fi) is just as important to plan for.
+
+## Offline options for Web Apps
+
+For Web Apps, there is really only one option when planning for an Offline Application and that is Service Workers.
+
+### Service Workers
+
```

Comment:
can we also add a link to the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API) for the Service Worker API?

Reply:
please disregard this, I see it's already in the resources section 👍